setup.py

```diff
@@ -1,13 +1,14 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
+import os
 import io
 import re
 from setuptools import setup
 
 with io.open('README.rst', 'rt', encoding='utf8') as f:
     readme = f.read()
 
-with io.open('cachelib/__init__.py', 'rt', encoding='utf8') as f:
+with io.open(os.path.join('cachelib', '__init__.py'), 'rt', encoding='utf8') as f:
```

Comment:
This one is not needed, since `/` works perfectly fine even on Windows.

Reply:
Thanks, you are right.
I have reverted this commit.